src/modules/flow/ipm/ipm.json

```diff
@@ -0,0 +1,475 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "ipm",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "Apache-2.0",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "output/ipm",
+      "description": "Inter-Processor Messaging boolean writer. Writes a boolean packet with a given ID to a different core.",
+      "methods": {
+        "open": "boolean_writer_open"
+      },
+      "name": "ipm/boolean-writer",
+      "options": {
+        "members": [
+          {
+            "data_type": "int",
+            "description": "ID of message. Readers will use this ID to receive messages. Must be greater than 0 and smaller than platform maximum id number.",
+            "name": "id"
+          }
+        ],
+        "version": 1
+      },
+      "in_ports": [
+        {
+          "data_type": "boolean",
+          "description": "Boolean packets sent here will be sent to another core.",
+          "methods": {
+            "process": "boolean_writer_process"
+          },
+          "name": "IN"
+        }
```

Comment:
why no `CONSUMED` (or `PROCESSED`) port? It could be used as a trigger to process further information and control the flow to not overflow the channel. Take a loop where the next iteration can be controlled by its own trigger (ie: accumulator), then you'd link that port to the accumulator increase.